googletest/src/gtest.cc

```diff
@@ -6130,14 +6134,16 @@ void UnitTestImpl::ListTestsMatchingFilter() {
 
     for (size_t j = 0; j < test_suite->test_info_list().size(); j++) {
       const TestInfo* const test_info = test_suite->test_info_list()[j];
-      if (test_info->matches_filter_) {
+      if (!test_info->matches_filter_)
+        continue;
+      if (!GTEST_FLAG_GET(list_tests_verbosely)) {
         if (!printed_test_suite_name) {
           printed_test_suite_name = true;
-          printf("%s.", test_suite->name());
+          printf("%s. ", test_suite->name());
```

Comment:
```suggestion
          printf("%s.", test_suite->name());
```

Reply:
@BMBurstein Thanks for catching the mistake.

Reply:
A wdisgne page or a note pad

Reply:
Besides that unless you have paths for it to go on theat it wont hold up to much is t even running

Reply:
It can ru of both sides but you have to be good